doc/releases/changelog-dev.md

```diff
@@ -4,7 +4,72 @@
 
 <h3>New features since last release</h3>
 
-* `DefaultQubit` devices now natively support parameter broadcasting.
+* The gradient transform `qml.gradients.param_shift` now accepts the new Boolean keyword
```

Comment:
Thanks :) 
I shifted the speedup remarks to the docstring and made the whole setup tape-based instead of QNode-based, which shortened it a bit more. Is this good or should I (re)move more stuff?

Reply:
Looking great :) Commented on one other example that could be moved to the docstrings, otherwise, I think this is quite succinct and onto the point. :+1: May have to be shrunk even more for the release, but that comes around when the entire picture for param broadcasting is available on the same page.